src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -144,11 +144,23 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
       // Throw this exception so it can be handled at the reconnection layer, and don't show the
       // error notification.
       throw ex;
-    } else {
+    } else if (!isNestedError(ex)) {
       showErrorNotification();
+    } else if (ex.innerErrors && ex.innerErrors.some(e => e.errorType === 'UnsupportedTransportError' && e.transport === HttpTransportType.WebSockets)) {
+      logger.log(LogLevel.Error, 'Unable to connect, please ensure you are using an updated browser that supports WebSockets.');
+    } else if (ex.innerErrors && ex.innerErrors.some(e => e.errorType === 'FailedToStartTransportError' && e.transport === HttpTransportType.WebSockets)) {
+      logger.log(LogLevel.Error, 'Unable to connect, please ensure WebSockets are available. A VPN or proxy may be blocking the connection.');
+    } else if (ex.innerErrors && ex.innerErrors.some(e => e.errorType === 'DisabledTransportError' && e.transport === HttpTransportType.LongPolling)) {
+      logger.log(LogLevel.Error, 'Unable to initiate a SignalR connection to the server. This might be because the server is not configured to support WebSockets. To troubleshoot this, visit https://aka.ms/blazor-server-websockets-error.');
     }
   }
 
+  // Check if the connection is established using the long polling transport,
+  // using the `features.inherentKeepAlive` property only present with long polling.
+  if ((connection as any).connection.features.inherentKeepAlive) {
```

Comment:
Spoke with @BrennanConroy regarding best means to identify if a connection is `Long Polling`. He suggested relying on this property implicitly (instead of explicitly adding a marker to the SignalR HttpTransport). Will be adding a E2E test that validates this behavior.

Reply:
I think we should log a warning, not an error, since the app will continue working.